browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -91,6 +105,120 @@ class MarkdownNoteDetail extends React.Component {
     this.updateNote(note)
   }
 
+  historyChecks () {
+    let back = this.state.backStack
+    if (back.past.length > 7) {
+      back.past.splice(0, 1)
+      this.setState({
+        backStack: back
+      }, () => {
+        this.save()
+      })
+    }
+    if (back.future.length > 7) {
+      back.future.splice(0, 1)
+      this.setState({
+        backStack: back
+      }, () => {
+        this.save()
+      })
+    }
+    if (back.past.length) {
+      if (!this.isBackActive) {
```

Comment:
This suppose to be `this.state.isBackActive`. Does it make sense?

Reply:
Yeah... my bad... Thank for your help @Rokt33r